core-bundle/src/Resources/contao/widgets/SerpPreview.php

```diff
@@ -109,20 +109,20 @@ private function getTitle(Model $model)
 	{
 		if (!isset($this->titleFields))
 		{
-			return $model->title;
+			return (string) $model->title;
```

Comment:
The description is nullable indeed, however, the title is not.

https://github.com/contao/contao/blob/8114301bc2ef7bedd90dcc5afc9ee634cd526dd2/core-bundle/src/Resources/contao/dca/tl_page.php#L229

So these changes have to be reverted.

Reply:
well not for `tl_page`. But you don't know all the tables SerpWidget is used on!

Reply:
This is a pointless discussion. The code inspector would detect and remove the unnecessary type cast anyway.

Reply:
IMHO we should fix this in lines 44 and 45:

```php
$title = StringUtil::substr((string) $this->getTitle($model), 64);
$description = StringUtil::substr((string) $this->getDescription($model), 160);
```

Reply:
well I was assuming the `getTitle()` and `getDescription()` should have typed return values at some point. And then they would need to cast the value.

Reply:
But even then, the return type would most likely be `string|null`, so one can distinguish between "not set" and "empty".